examples/echo_bot.py

```diff
@@ -33,7 +48,7 @@ def main():
 
 class EchoBot(KikClientCallback):
     def __init__(self):
-        self.client = KikClient(self, username, password)
+        self.client = KikClient(self, BOT_USERNAME, BOT_PASSWORD, BOT_NODE_JID)
```

Comment:
I think it's a little confusing that `BOT_USERNAME` is capital (it makes you think it's a hard-coded constant).
Change it to back to lower case?

Reply:
Sure thing, reverted this [here](https://github.com/tomer8007/kik-bot-api-unofficial/pull/224/commits/a057c135f78e7592a981d940b707c0558208fb58).